added URL for server back into code

picklesrein opened the pull request:
Version 0.0.4 had the server URL imbedded in the report which I find useful. However it was generated from the hostname which sometimes produced erroneous URLs. Instead this update pulls the URL directly from the server's "$Conf{CgiURL} = '';" value.

picklesrein committed:
added URL for server back into code

Version 0.0.4 had the server URL imbedded in the report which I find useful. However it was generated from the hostname which sometimes produced erroneous URLs. This update pulls the URL directly from the server's "$Conf{CgiURL} = '';" value instead.

picklesrein committed:
increment version number

picklesrein committed:
updated path prefix to match common installation

moisseev commented on BackupPC_report:
```diff
@@ -45,6 +47,9 @@ Please su [-m] $bpcUser first.\n"
   or die "BackupPC::Lib->new failed\n";
 
+%Conf = $bpc->Conf(); 
+my $SvrURL = $Conf{CgiURL};    # set URL
```
Why do we need to introduce another variable?

moisseev commented on BackupPC_report:
```diff
@@ -8,7 +8,7 @@
 use strict;
 use warnings;              # global flag -w produces warnings from BackupPC::Lib
-use lib '/usr/local/lib';  # <<< change path prefix to match your installation
+use lib '/usr/share/BackupPC/lib';  # <<< change path prefix to match your installation
```
I understand why you did this, but I am not agree with this change just because it is not convenient for me since I am a FreeBSD user.
Probably the proper way to resolve this issue is to include the `BackupPC_report` script into `BackupPC` code.

moisseev reviewed (changes requested):
> Version 0.0.4 had the server URL imbedded in the report

Actually, I don't remember that. More over, I can't find it in git log, but I like the idea. Thank you.

moisseev commented on BackupPC_report:
```diff
@@ -14,6 +14,8 @@ use POSIX qw( getuid getgid setuid setgid floor );
 use Getopt::Long;
 use Pod::Usage;
 
+use vars qw(%Conf);
```
Can we use `our` declaration instead?

picklesrein replied:
implemented changes

picklesrein committed:
removed flavor-dependent path and extra variable

Backed out changes on library path, removed extra variable and utilized 'our' declaration.

picklesrein commented:
I believe requested changes have been committed.
Thank you!

moisseev commented:
@picklesrein , thank you!
I am sorry, I was wrong. Actually we don't need to declare global variable at all as we have already got the configuration options here:
https://github.com/moisseev/BackupPC_report/blob/59544f31d34c71cb8d3631008bf487baf0b602a3/BackupPC_report#L45

So, all that we need is to add just one line:
```perl
# Print BackupPC URL
printf "\nURL: $bpc->{Conf}{CgiURL}\n\n";
```

picklesrein committed:
removed unnecessary declarations; modified printf

picklesrein commented:
updated

moisseev commented:
Thank you again!
